core/utils/assets-config-provider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Elementor\Core\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Assets_Config_Provider extends Collection {
+	/**
+	 * Load asset config from a file into the collection.
+	 *
+	 * @param $key
+	 * @param $path
+	 *
+	 * @return $this
+	 */
+	public function load( $key, $path ) {
+		if ( ! file_exists( $path ) ) {
+			return $this;
```

Comment:
Don't you want to return an indication that something went wrong? return false or something? (same as for the condition below)

Reply:
I think about throwing an exception but I am not sure about it. let's talk about it.